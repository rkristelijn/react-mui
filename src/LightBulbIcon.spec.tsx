import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';

import LightBulbIcon from './LightBulbIcon';

// todo: it runs, but this seems untestable for now, so mocking
jest.mock('./LightBulbIcon', () => 'LightBulbIcon');

describe('LightBulbIcon', () => {
  let props;
  it('should initiate', () => {
    expect(LightBulbIcon).toBeTruthy();
  });

  it('should match snapshop', () => {
    props = { 'aria-busy': 'false' };
    const component = renderer.create(<LightBulbIcon sx={props} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
