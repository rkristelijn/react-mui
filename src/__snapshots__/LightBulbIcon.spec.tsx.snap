// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LightBulbIcon should match snapshop 1`] = `
<LightBulbIcon
  sx={
    Object {
      "aria-busy": "false",
    }
  }
/>
`;
